test/uk/gov/hmrc/economiccrimelevyreturns/forms/behaviours/CurrencyFieldBehaviours.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyreturns.forms.behaviours
+
+import play.api.data.{Form, FormError}
+
+trait CurrencyFieldBehaviours extends FieldBehaviours {
+
+  def currencyField(form: Form[_], fieldName: String, nonNumericError: FormError): Unit =
```

Comment:
The following tests would be also useful.

Mixture of spaces before, beginning and after number/decimal, etc